src/main/java/uk/ac/ebi/eva/evaseqcol/utils/JSONStringListExtData.java

```diff
@@ -13,6 +13,29 @@ public JSONStringListExtData(List<String> object) {
         super(object);
     }
 
+    /**
+     * The same as the Overridden toString method
+     * Used to avoid code duplication in different classes
+     * // TODO: We can find a better way to avoid code duplication*/
+    public static String toString(List<String> object) {
```

Comment:
I'm not sure how well and efficient this approach is (to make to methods with the same code but with different signatures). Doesn't seem too good.

Reply:
Couple of things here:
 - Could change the name of the method because it convert a list of string to a string rather than convert the current object to String
 - There are still some duplication with this method implemented (in part) in the [JSONExtData](https://github.com/waterflow80/eva-seqcol/blob/4ee147018e2624ca0529017665d2f56b453e20a7/src/main/java/uk/ac/ebi/eva/evaseqcol/utils/JSONExtData.java#L38]) ? 
 - You should be  able to call the static method from the other `toString()` and pass the [object](https://github.com/waterflow80/eva-seqcol/blob/4ee147018e2624ca0529017665d2f56b453e20a7/src/main/java/uk/ac/ebi/eva/evaseqcol/utils/JSONExtData.java#L14C26-L14C32) as long as it is protected instead of private.

Reply:
Can't we just do `new JSONStringListExtData(object).toString()` instead of having an additional static method?
If we do still need it, I agree it should be called something besides `toString`...